ow_lander/src/ow_lander/server.py

```diff
@@ -12,13 +14,23 @@ class ActionServerBase(ABC):
   and logging for all OceanWATERS actions.
   """
 
+  ACTION_GOAL_STATUS_TOPIC = "/action_goal_status"
+  # allocate and initialize the container for goal status msgs
+  _goal_status_array = []
+  for _ in range(owl_msgs.msg.ActionGoalStatus.NUM_GOAL_TYPES):
+      _goal_status_array.append(actionlib_msgs.msg.GoalStatus())
```

Comment:
@AstroStucky I feel like having initialization code outside of __init__ is strange, but I'm not seeing any other python way to enforce the size/structure of the shared class-member array. Thoughts?

Reply:
I believe this is fine to do. You wouldn't want to use `__init__` because that initializes and instance, but here you want to initialize a class member, and that should only happen once, ideally when `ActionServerBase` is imported.

This could be shortened to a single line if you'd like.
```Python
  _goal_status_array = [actionlib_msgs.msg.GoalStatus() for _ in range(owl_msgs.msg.ActionGoalStatus.NUM_GOAL_TYPES)
```